app/src/main/java/com/project/gamedb/ui/popular/PopularPresenter.kt

```diff
@@ -0,0 +1,15 @@
+package com.project.gamedb.ui.popular
+
+import com.project.gamedb.data.source.remote.RemoteDataRepository
+
+class PopularPresenter(
+    private val view: PopularContract.View,
+    private val repository: RemoteDataRepository
+) : PopularContract.Presenter {
+    override fun getMainGameFeature() {
+    }
```

Comment:
nếu header class nhiều dòng, để cách 1 dòng trống 
https://kotlinlang.org/docs/coding-conventions.html#class-layout
```suggestion
    private val repository: RemoteDataRepository
) : PopularContract.Presenter {

    override fun getMainGameFeature() {
    }
```